src/pages/imprint.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { HelmetSeo } from '~/components/Helmet/HelmetSeo';
+import { LayoutArticle } from '~/components/Layout';
+import { ExternalLink } from '~/components/Links/ExternalLink';
+import BmdvFundingImage from '~/static/BMDV_Foerderung.svg';
+
+const ImprintPage = () => {
+  return (
+    <LayoutArticle>
+      <HelmetSeo noindex title="Impressum" />
+      <h2>Impressum</h2>
```

Comment:
```suggestion
      <h1>Impressum</h1>
```

Lets start with `h1` and go down from there. I did not comment on the other `hx` below.